src/Mod/JtReader/App/JrJt/Context.h

```diff
@@ -36,7 +36,7 @@ class Context
     uint8_t VersionMinor;
     uint8_t VersionMajor;
 
-    bool Endiness;
+    bool Endianness;
```

Comment:
Please review

Reply:
Changing the name must also lead to further changes in the code where the bool is used. If the bool is not used, it should be removed. I only see that you changed the name, nothing else. Is that right?

Reply:
Confirmed, searching the whole codebase (again) shows this is the only place this bool is used.

Reply:
Removed in [f34b3c0](https://github.com/FreeCAD/FreeCAD/pull/7919/commits/f34b3c0ecf682743d613da460d48b0fe1b6566b7)